frontend/index.mjs

```diff
@@ -0,0 +1,76 @@
+export default async function (plop) {
+  plop.setGenerator("frontend:init", {
+    description: "Init a FE with general configuration",
+    prompts: [
+      {
+        type: "input",
+        name: "criName",
+        message: "CRI Name?",
+      },
+      {
+        type: "input",
+        name: "mainRoute",
+        message: "URL path for the main route?",
+      },
+      {
+        type: "checkbox",
+        name: "initOptions",
+        message: "What would you like to initialise?",
+        choices: [
+          { name: ".dotfiles", value: "dotfiles" },
+          { name: "npm", value: "npm" },
+          { name: "Express Webserver", value: "express" },
+          { name: "Cucumber", value: "cucumber" },
+        ],
+      },
+    ],
+    actions: function (data) {
+      const actions = [];
+
+      if (data.initOptions.includes("dotfiles")) {
+        actions.push({
+          type: "addMany",
+          destination: "./",
+          base: "templates",
+          templateFiles: ".*",
+          globOptions: { dot: true },
+          verbose: true,
+        });
+      }
+
+      if (data.initOptions.includes("npm")) {
+        actions.push({
+          type: "add",
+          path: "package.json",
+          templateFile: "templates/package.json.hbs",
+          verbose: true,
+          skipIfExists: true,
+        });
+      }
+
+      if (data.initOptions.includes("express")) {
+        actions.push({
+          type: "addMany",
+          destination: "src",
+          base: "templates/express/src",
+          templateFiles: "**",
+          globOptions: { dot: true },
+          verbose: true,
+        });
+      }
+
+      if (data.initOptions.includes("cucumber")) {
+        actions.push({
+          type: "addMany",
+          destination: "tests/browser",
```

Comment:
This is a minor one, most of the frontend have test folder - here we have tests. I suppose tests is fine

Reply:
Should there be a `README.md` here as well

Reply:
@jkunle That's a good question - what do we want our standard to be? Some repos have `test`, some have `tests`.

Reply:
- `README.md` across various directories have been added
- `LICENSE` is currently generated when creating a repo.
- `.env` across the root and `test/browser` folders have been added
- `Dockerfile`, `local.Dockerfile` for the root have been added

Reply:
`test` vs `tests` has been decided by the team in the favour of `tests`